### ISSUE TITLE
Wrong and redundant portal

### ISSUE THREAD

**Issue by al-wei** [Jboymercs/nether_backport#18]
![image](https://github.com/user-attachments/assets/057a32f7-d00d-48e9-b2d4-5ed2c75847a1)

**Comment by Jboymercs** [comment]
Yup, this issue still occurs. Even though we stated we are ceasing development on this, we'll still probably fix it cause it annoys me personally since I tried to fix this last update lol. Thank you for reporting!

**Comment by Jboymercs** [comment]
Should be fixed in 0.5, I'll re-open if I encounter it again or someone else has it happen. Thank you for reporting!
